.gitignore

```diff
@@ -1,4 +1,5 @@
 target
 Cargo.lock
 !diesel_cli/Cargo.lock
-.env
\ No newline at end of file
+.env
+.vscode
```

Comment:
This should live in your own global `.gitignore` config, not in the diesel specific one.